patternlab/styleguide/source/_patterns/02-molecules/footer-links.twig

```diff
@@ -1,14 +1,14 @@
 <section class="ma__footer-links">

```

Comment:
Please remove this `<section>` which is an extra wrapper and not serving anything.  It has no effect for styling either.